### ISSUE TITLE
Update for go module

### ISSUE THREAD

**Issue by HT808s** [Arman92/go-tdlib#38]
First of all, great project and great job !

I propose to add [GO modules](https://blog.golang.org/using-go-modules) related file to the directory

I suggest to set GO minimal required version to `1.13`

I strongly advise starting to tag this project by following the [semver](https://semver.org/) convention and with the format `v0.0.1`

That will make your library compliant with go modules and will provide users of your library better control over the versions they want to use

WDYT @Arman92 ?

**Comment by Arman92** [comment]
Yes, sure it is a good idea, I will migrate to go mods and also set tags
